File: src/main/java/org/example/book/greedy/거스름돈.java
package org.example.book.greedy;


public class 거스름돈 {
    public static void main(String[] args) {
        int n = 1260;
        int count = 0;
        int[] coin_types = {500, 100, 50, 10};

        for (int coin : coin_types) {
            count += n / coin;
            n %= coin;
        }
        System.out.println(count);
    }
}

File: pythonAlgorithm/book/그리디/거스름돈.py
def solution(n):
    coin_types = [500,100,50,10]
    count = 0
    for coin in coin_types:
        count += n // coin
        n%=coin
    return count

if __name__ == '__main__':
    n = 1260
    answer = solution(n)
    print(answer)